var/spack/repos/builtin/packages/r-packrat/package.py

```diff
@@ -11,9 +11,9 @@ class RPackrat(RPackage):
     and reproducible way."""
 
     homepage = "https://github.com/rstudio/packrat/"
-    url      = "https://cloud.r-project.org/src/contrib/packrat_0.4.7-1.tar.gz"
-    list_url = "https://cloud.r-project.org/src/contrib/Archive/packrat"
+    cran = "packrat"
 
+    version('0.7.0', sha256='e8bce1fd78f28f3a7bf56e65a2ae2c6802e69bf55466c24e1d1a4b8a5f83dcc2')
     version('0.5.0', sha256='d6a09290fbe037a6c740921c5dcd70b500e5b36e4713eae4010adf0c456bc5f7')
```

Comment:
With the change in download source, are you able to `spack stage r-packrat@0.5.0`?

Reply:
Yes after clearing the cache `spack stage r-packrat@0.5.0' works